packages/mobile/src/components/challenge-rewards-drawer/ChallengeRewardsDrawerProvider.tsx

```diff
@@ -84,7 +84,7 @@ export const ChallengeRewardsDrawerProvider = () => {
     audioToClaim = challenge.claimableAmount
     audioClaimedSoFar = challenge.disbursed_amount
   } else if (challenge?.state === 'completed') {
-    audioToClaim = challenge.totalAmount
+    audioToClaim = challenge.claimableAmount
```

Comment:
pretty sure this was there for a reason. can you please double check to make sure this change is correct. because, unless the upstream challenge logic has changed, I doubt it is.

Reply:
yea good call, i was confused why totalAmount was used. turns out claimableAmount is 0 for non cooldowns on completion. 

my changes worked for cooldown but not non-cooldown same with the other comment

Reply:
nice thx